### PR TITLE
Issue #1388 Adding Minishift version information to debug logs

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

File: cmd/minishift/cmd/root.go [hunk 1]
```diff
@@ -141,6 +140,11 @@ var RootCmd = &cobra.Command{
 		}
 
 		setDefaultActiveProfile()
+
+		// Adding minishift version information to debug logs
+		if glog.V(2) {
+			fmt.Println(fmt.Sprintf("-- minishift version: %s", version.GetMinishiftVersion()))
```
Comment (budhrg) [inline cmd/minishift/cmd/root.go]: Should be `"-- Minishift version: %s"` as `minishift` (small letter) indicates command.

File: cmd/minishift/cmd/root.go [hunk 2]
```diff
@@ -141,6 +140,11 @@ var RootCmd = &cobra.Command{
 		}
 
 		setDefaultActiveProfile()
+
+		// Adding minishift version information to debug logs
+		if glog.V(2) {
+			fmt.Println(fmt.Sprintf("-- minishift version: v%s+%s", version.GetMinishiftVersion(), version.GetCommitSha()))
```
Comment (budhrg) [inline cmd/minishift/cmd/root.go]:
```
$ minishift version
minishift v1.11.0+d219f961

$ minishift start -v 3
-- minishift version: v1.11.0+d219f961
-- Starting profile 'minishift'
[...]
```

Though I am still not sure if we should use `Minishift version: v1.11.0+d219f961`. WDYT @praveenkumar ?

Comment (praveenkumar) [reply cmd/minishift/cmd/root.go]: It's binary version for `Minishift` project so I am fine with the current implementation.

Comment (praveenkumar) [reply cmd/minishift/cmd/root.go]: @budhrg It's binary version for Minishift project so I am fine with the current implementation.

Comment (praveenkumar) [inline cmd/minishift/cmd/root.go]: 👍

**Pull request merged**